src/librustc_save_analysis/lib.rs

```diff
@@ -228,6 +232,7 @@ impl<'l, 'tcx: 'l> SaveContext<'l, 'tcx> {
                     visibility: From::from(&item.vis),
                     docs: docs_for_attrs(&item.attrs),
                     sig: self.sig_base(item),
+                    attributes: remove_docs_from_attrs(&item.attrs),
```

Comment:
could the remove_docs_from_attrs call be moved to the lower function so it is not duplicated?